src/tribler-core/tribler_core/modules/metadata_store/tests/test_channel_download.py

```diff
@@ -21,6 +21,8 @@ def setUpPreSession(self):
         super(TestChannelDownload, self).setUpPreSession()
         self.config.set_chant_enabled(True)
         self.config.set_libtorrent_enabled(True)
+        self.config.set_libtorrent_dht_readiness_timeout(0)
```

Comment:
Do you need to set this to `0` if the DHT is already disabled?

Reply:
Probably yes, since `set_libtorrent_readiness_timeout` knows nothing about if DHT is enabled or not.